src/components/GEM2SLoadingScreen.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import {
+  Result, Button, Progress, Row, Col, Typography, Space,
+} from 'antd';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+const { Title, Text } = Typography;
+
+const GEM2SLoadingScreen = ({ pipelineStatus, completedSteps, steps }) => {
```

Comment:
This component is mostly the same as `PipelineRedirectToDataProcessing`, apart from the specific message strings and other small details. Have you looked at reusing that component? Or make a new component called `Bac kendLoadingScreen` and use it for both gem2s and data processing?

Reply:
Yes, actually it's very similar. I'll try to refactor that component to make it cover both use cases